src/Field/ShortText.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+// vim:ts=4:sw=4:et:fdm=marker:fdl=0
+
+namespace atk4\data\Field;
+
+use atk4\data\Field;
+use atk4\data\ValidationException;
+
+/**
+ * Basic string field type. Think of it as field type "string" in past.
+ *
+ * Most of the time this is most basic type you will use.
+ */
+class ShortText extends Text
```

Comment:
I suggest we have two classes:

Line (may be a short or a long line, but enter is not permitted)
Text (may be a short or a long text, will contain some paragraphs)

Reply:
Addressed.
Simply renaming ShortText class to Line does this :)
No more need for _short_ vs _long_. Limit can be defined by max_length property in both of these classes.